tasks/eth/006-MCP-L1/VALIDATION.md

```diff
@@ -53,239 +61,271 @@ And we do indeed see these entries:
 **Notes:**
 - The value `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc` occurs
   multiple times below, and corresponds to the storage key of the implementation address as defined
-  in [Proxy.sol](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/src/universal/Proxy.sol#L104) and [Constants.sol](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/src/libraries/Constants.sol#L26-L27).
+  in [Proxy.sol](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/src/universal/Proxy.sol#L104) and [Constants.sol](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/src/libraries/Constants.sol#L26-L27). This is useful for EIP-1967 proxies.
+- Check the provided links to ensure that the correct contract is described at the correct address. The superchain registry is the source of truth for contract addresses and etherscan is supplementary.
 
 ### `0x229047fed2591dbec1ef1118d64f7af3db9eb290` (`SystemConfigProxy`)
 
+Links:
+- [Etherscan](https://etherscan.io/address/0x229047fed2591dbec1ef1118d64f7af3db9eb290)
+- [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/52d3dbd1605dd43f419e838584abd0ec163d462b/superchain/extra/addresses/mainnet/op.json#L10)
+
+State Changes:
+
+Please ensure that each link to the `superchain-registry` correctly corresponds to OP Mainnet as the superchain registry contains data for
```

Comment:
I added links to etherscan that can cross check that the addresses in the registry are correct, meaning that is the contract that we say it is. Right now I am unaware of any sort of merge protections against adding in improper addresses, although this could be added as part of CI, noted in https://github.com/ethereum-optimism/security-pod/issues/99